userfiles/civet_output/views/_surface_viewer.html.erb

```diff
@@ -130,7 +130,7 @@
 
 <script type="text/javascript">
 
-  $("#surface-viewer").load("https://brainbrowser.cbrain.mcgill.ca/surface-viewer-widget?version=2.3.0&height=600&width=600&viewer_callback=init");
+  $("#surface-viewer").load("https://brainbrowser.cbrain.mcgill.ca/surface-viewer-widget?version=2.4.0&height=600&width=600&viewer_callback=init");
```

Comment:
It would be nice if this wasn't hardcoded... there might be a place in a Rails app for setting arbitrary configuration key/values.